docs/en/stack/ml/machine-learning-intro.asciidoc

```diff
@@ -0,0 +1,49 @@
+[chapter,role="xpack"]
+[[machine-learning-intro]]
+= What is Elastic {ml-app}?
+:keywords: {ml-init}, {stack}
+:description: An introduction to the breadth of Elastic {ml-features}.
+
+{ml-cap} features analyze your data and generate models for its patterns of
+behavior. The type of analysis that you choose depends on the questions or
+problems you want to address and the type of data you have available.
+
+[discrete]
+[[machine-learning-unsupervised]]
+== Unsupervised {ml}
+
+There are two types of analysis that can deduce the patterns and relationships
+within your data without training or intervention: _{anomaly-detect}_ and
+_{oldetection}_.
+
+<<ml-ad-overview,{anomaly-detect-cap}>> requires time series data. It constructs a
+probability model and can run continuously to identify unusual events as they
+occur. The model evolves over time; you can use its insights to forecast future
+behavior.
+
+<<ml-dfa-finding-outliers,{oldetection-cap}>> does not require time series data; 
```

Comment:
I think it's a great idea to split the different types of analyses in terms of being supervised or unsupervised. However, we have another layer that we use to organize ML content and the TOC reflects that logic (anomaly detection/data frame analytics). For this reason, I think we should mention in the short description of outlier detection, classification, and regression that those are different types of data frame analytics. Something like:
```suggestion
<<ml-dfa-finding-outliers,{oldetection-cap}>> is a type of <<ml-dfanalytics,{dfanalytics}>> that does not require time series data; 
```
What do you think?